crates/nu-protocol/src/errors/shell_error.rs

```diff
@@ -595,6 +595,19 @@ pub enum ShellError {
         src_span: Span,
     },
 
+    /// Not all rows contain data in the requested column.
+    ///
+    /// ## Resolution
+    ///
+    /// Use the Optional Operator, --ignore-errors with get or select, or fill in the missing values.
```

Comment:
This comment only appears in the Rust crate documentation. If you want to show it you can use the `miette` `#[help]` attribute.

As `follow_cell_path` and friends are also used in a bunch other commands , you should check that the error message like that makes universal sense, including its span annotations.